www/js/lib/uiUtil.js

```diff
@@ -415,9 +415,12 @@ define(rqDef, function() {
      * For each contentTheme, a stylesheet must be provided in www/css that is named 'kiwixJS' + contentTheme
      * A rule may additionally be needed in app.css for full implementation of contentTheme
      * 
-     * @param {String} theme The theme to apply (light|dark[_invert|_mwInvert])
+     * @param {String} theme The theme to apply (light|dark[_invert|_mwInvert]|auto[_invert|_mwInvert])
      */
     function applyAppTheme(theme) {
+        var darkPreference = window.matchMedia('(prefers-color-scheme:dark)');
+        var selectedThemeContent = theme.replace(/^[^_]*/, '');
+        if (/^auto/.test(theme)) theme = darkPreference.matches ? 'dark' + selectedThemeContent : 'light' + selectedThemeContent;
```

Comment:
So, instead of setting an invalid value here, why not just set the last value in the ternary construct to `'light'` instead of `'light' + selectedThemeContent`?